assets/customer.css

```diff
@@ -444,6 +446,10 @@
 }
 
 @media screen and (min-width: 750px) {
+  .account a svg {
+    margin-bottom: -0.03rem;
```

Comment:
I feel like `-0.3rem` looks a bit nicer on both mobile and desktop. Just a thought but it's a very minimal difference to it: [video](https://screenshot.click/21-08-9aodn-zwnup.mp4)